mailbox/cassandra/src/test/java/org/apache/james/mailbox/cassandra/mail/CassandraMailboxMapperTest.java

```diff
@@ -312,4 +316,17 @@ public void hasChildrenShouldReturnChildWhenExistsInV2Table() {
     
         assertThat(hasChildren).isTrue();
     }
+
+    @Test
+    public void findMailboxWithPathLikeShouldUnionBothVersions() {
```

Comment:
can you also add a test for hasChildren please?

Reply:
hasChildrenShouldReturnChildWhenExistsInBothTables do test this, and we have tests with the children being in V1, V2 so I'd consider this tested.